extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/LDAPGuacamoleProperties.java

```diff
@@ -176,32 +176,22 @@ private LDAPGuacamoleProperties() {}
     };
 
     /**
-     * Whether or not we should follow referrals.
+     * A time limit on LDAP operations.
      */
-    public static final BooleanGuacamoleProperty LDAP_FOLLOW_REFERRALS = new BooleanGuacamoleProperty() {
-
-        @Override
-        public String getName() { return "ldap-follow-referrals"; }
-
-    };
-
-    /**
-     * Maximum number of referral hops to follow.
-     */
-    public static final IntegerGuacamoleProperty LDAP_MAX_REFERRAL_HOPS = new IntegerGuacamoleProperty() {
```

Comment:
Is this not supported by the Apache Directory LDAP API?

Reply:
Not that I can tell - let me know if you can find something on it, but I'm unable to find any details on how to do it.